internal/adapters/cloud/aws/ec2/ec2.go

```diff
@@ -53,7 +53,7 @@ func (a *adapter) Adapt(root *aws2.RootAdapter, state *state.State) error {
 		return err
 	}
 
-	state.AWS.EC2.DefaultVPCs, err = a.getDefaultVPCs()
```

Comment:
Previously, we were only interested in the default VPCs, now we have checks that involve all VPCs, we need to change the state object to have a slice of VPCs